src/modules/launcher/Plugins/Microsoft.Plugin.Uri/UriHelper/ExtendedUriParser.cs

```diff
@@ -18,6 +18,7 @@ public bool TryParse(string input, out System.Uri result)
             }
 
             // Handle common cases UriBuilder does not handle
+            // Using Ordinal since these are internal and used with symbols
             if (input.EndsWith(":", StringComparison.Ordinal)
```

Comment:
This should be currentculture since `input` is a user typed string. @alekhyareddy28 thoughts?

Reply:
Thanks @somil55. Could you please clarify why - my thinking was that the method here is only comparing the end of string with ":" - doesn't that make it a more appropriate case for Ordinal?

Reply:
They should give the same output but there is a possibility of some corner cases, where this could fail. So `Ordinal` would be a more robust solution.

Reply:
Got it, thanks! changing this to `CurrentCulture`